contributing/getting-started-e2e.md

```diff
@@ -368,10 +368,10 @@ Note: You can automatically run critical checks before `git push` by enabling ou
 To run the tests that are affected by the changes on your feature branch:
 
 ```
-gulp test --local-changes
+gulp unit --local_changes
 ```
 
-By default, all tests are run on Chrome. Pass `--firefox` or `--safari` to run tests in Firefox and Safari, respectively.
+By default, all tests are run on Chrome. Pass `--firefox`, `--safari`, `--edge`, or `--ie` to run tests on other browsers.
```

Comment:
Nit: is this accumulative or should be mutually exclusive? (what happens if I run `gulp unit --firefox --edge`?)

Reply:
Mutually exclusive, it will use the first browser that matches (Edge in this case). Clarified the wording!